lightning/src/ln/interactivetxs.rs

```diff
@@ -396,9 +396,14 @@ impl InteractiveTxSigningSession {
 	/// unsigned transaction.
 	pub fn provide_holder_witnesses(
 		&mut self, channel_id: ChannelId, witnesses: Vec<Witness>,
```

Comment:
Predates the PR,  but looks like there is an unnecessary `into_iter().collect()` on `witnesses`.